src/propsheet/registry.cpp

```diff
@@ -169,6 +169,7 @@ DWORD GetRegistryValues(
 
     // initial values for global v2 settings
     g_fForceV2 = GetConsoleBoolValue(CONSOLE_REGISTRY_FORCEV2, TRUE);
+    g_fLaunchedAsV2 = g_fForceV2;

```

Comment:
wait, is this always set even for the shortcut case? I'm alarmed that it's set here but used in SaveConsoleSettingsIfNeeded's _link_ path

Reply:
Yea, because the ForceV2 setting isn't per-registry or per-shortcut, it's _always_ in the registry root, as a global-only setting.

(I'm leaving this open to ensure that it works for the "launch straight into the shortcut, no conhost" path)

Reply:
This does in fact work φ(*￣0￣)